### ISSUE TITLE
Confidence score

### ISSUE THREAD

**Issue by golunov** [patrikhuber/superviseddescent#24]
Hi, Patrik!

I'm fascinated by your work!

Did you see confidence score in Intraface face alignment? Is it possible to implement it here?

I'll appreciate your answer.
Thanks.

**Comment by datangguo** [comment]
I think the confidence score will be useful during facial feature tracking, so the app know when to set have_face = 0 during tracking and start to reinitialising :) any idea for it? @patrikhuber

**Comment by patrikhuber** [comment]
Hi,

Thanks!

Intraface do not describe how they do the confidence. We tried stuff like train a classifier using the HOG features from training or a regressor using the distance measure to the ground truth, and it all kind of worked, but I did not have the time to fully train it. If it's of interest to you, it's rather easy to train though. If you want to do it with a classifier, you'll need to hook up superviseddescent to something like liblinear, libsvm, or tiny-cnn. But just using the regression framework in superviseddescent also didn't work too bad.
